### ISSUE TITLE
training loss not decrease

### ISSUE THREAD

**Issue by ccl-private** [BR-IDL/PaddleViT#39]
I train the landcover dataset in the image segment model you provide, but the loss does not decrease.
![image](https://user-images.githubusercontent.com/36214675/138221247-1c3aa0d1-8adf-4648-9b09-059665cbb323.png)

**Comment by xperzy** [comment]
@wutianyiRosun , Could you please help with this issue?

**Comment by GuoQuanhao** [comment]
> I tried segmenter before, trained one night on two RTX3090 cards, and the loss still did not decrease. why does this happen?

Hello, we currently make the performance of the model reach the SOTA, But the current segmentation training process is not complete, the loss func and augmentation and other solvers are in progress. You can follow my [aistudio](https://aistudio.baidu.com/aistudio/projectdetail/2534055). I add some loss function and other tricks, for Trans2seg, my loss is from 5.4 to 0.02 for total training progress(16000steps), for DPT, from 5.4 to 2.0 for 18850/1600000(steps)
